mobile/lib/features/trade/trade_theme.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class TradeTheme extends ThemeExtension<TradeTheme> {
+  // Unfortunately shades of Colors in flutter are not constant; see https://github.com/flutter/flutter/issues/31351
+  // Workaround to make is constant: Initialize the color from HEX
+  // For color codes see: https://api.flutter.dev/flutter/material/Colors/green-constant.html
+  static const Color green600 = Color(0xFF43A047);
+  static const Color red600 = Color(0xFFE53935);
```

Comment:
Random thought: we might want to document these colors outside of the code base so that we also use them elsewhere.

Reply:
yes, I think we should eventually store this "somewhere" globally. I don't have a clear vision on what the best place is yet and how far we should go. One idea I have is to define a design system that is available publicly online (we had this for COMIT at some point but I can't find the page where we managed this), and then pull all the information from there into the code / Figma (...). 
We can also store them in the `Brand Assets` in Notion.

For now I'd say:

We will have to align the colors that we use in Figma with those in the MVP. If that means that we adapt some of the Figma colors to Material codes, or vice-versa I leave open. 

Once the MVP is shipped the code becomes law, and changes to colors in Figma should always be reflected in code.
Let's come back to this once we are closer to having a more complete color pallet.